python_dwd/data_storing.py

```diff
@@ -37,18 +38,14 @@ def store_dwd_data(station_data: pd.DataFrame,
     request_string = _build_local_store_key(
         station_id, parameter, time_resolution, period_type)
 
-    local_filepath = build_local_filepath_for_station_data(folder)
+    local_filepath: Union[Path, str] = build_local_filepath_for_station_data(folder)
```

Comment:
i have never seen sth. like that. type hint the output in the  build_local_filepath_for_station_data function is enough

Reply:
Improved.